packages/@tinacms/cli/src/next/commands/baseCommands.ts

```diff
@@ -106,7 +109,7 @@ export abstract class BaseCommand extends Command {
         })
         warnings.push(...res.warnings)
       },
-      text: 'Indexing local files',
```

Comment:
@logan-anderson why are we distinguishing in the status text between 'local files' and 'self-hosted content'. It seems like in both cases, the content is local. Isn't it more about the destination? Like 'Indexing content to local data layer' & 'Indexing content to self-hosted data layer'?

Reply:
@kldavis4 I like that better! ill update it

Reply:
Just updated to "Indexing to self-hosted data layer" and "Indexing local content" what do you think?

Reply:
sounds good